### ISSUE TITLE
[Tests] - Controllers  

### ISSUE THREAD

**Issue by allanmessias** [allanmessias/jobber#3]
In order to test my Controllers, I need to do some mocking functions and objects to make it work. 

During some my research of doing this, I've came across these stackoverflow links:
https://stackoverflow.com/questions/57964299/mocking-express-request-with-jest-and-typescript-using-correct-types
https://stackoverflow.com/questions/55463886/unit-testing-controllers-use-jest-nodejs
https://stackoverflow.com/questions/54315266/jest-how-to-test-express-api-post-request

Basically, I need to mock my request with some data, and do some conditional checking to see if the property I'm looking for is there.

**Comment by allanmessias** [comment]
I found a lib to work with [ngrok](https://www.npmjs.com/package/ngrok) and make the tests more automated and to fetch my domain tunnel every time I reset it. To do so, I need to fetch my tunnel, using a lib's function that's connected to the [Ngrok Api](https://ngrok.com/docs/api).
